### ISSUE TITLE
OpenTracing and OpenCensus have merged to form OpenTelemetry!

### ISSUE THREAD

**Issue by agarbutt** [newrelic/java-aws-lambda#28]
Should we begin to leverage this library instead?

https://github.com/open-telemetry/opentelemetry-java

What changes if any, need to happen for New Relic?

**Comment by jkwatson** [comment]
Hi @agarbutt !  New Relic also has a (beta!) OpenTelemetry exporter for metrics and spans https://github.com/newrelic/opentelemetry-exporter-java. I'm not sure if there's any AWS lambda instrumentation for OTel yet, but if you want to try it out, we'd love the feedback.

**Comment by agarbutt** [comment]
@jkwatson Ill let you know, i am implementing an AWS API Gateway WebSocket service with Lambda implemented in Java. 

Ill take a look.

**Comment by XiXiaPdx** [comment]
Hi @agarbutt , I'm guessing you are unblocked and have moved on with your AWS Lambda project. I'm going to go ahead and close this issue. Please feel free to open it again or submit a new issue if we can assist further!
